src/Shell/Task/PluginTask.php

```diff
@@ -235,7 +237,7 @@ protected function _modifyAutoloader($plugin, $path)
             $cwd = getcwd();
 
             // Windows makes running multiple commands at once hard.
-            chdir(dirname($path));
+            chdir(ROOT);
```

Comment:
Please consider testing - you _don't_ want to be modifying `ROOT . DS . 'composer.json` when tests run and that's why it's in a separate function (and why generally I purged constants from the code recently).